tests/test_core.py

```diff
@@ -22,7 +22,8 @@ def sim_fixt():
            "Clock": {"start_time": 0,
                      "end_time": 10,
                      "num_steps": 100},
-           "Tools": {"ExampleTool": {"type": "ExampleTool", "custom_name": "example"}},
+           "Tools": {("ExampleTool", "example"): {"type": "ExampleTool", "custom_name": "example"},
+                     ("ExampleTool", "example2"): {"type": "ExampleTool", "custom_name": "example2"}},
```

Comment:
See note in review. You can should use a list if you want multiple tools of the same type. Also, you don't need to specify "type". That gets set automatically.
```suggestion
           "Tools": {"ExampleTool": [
                           {"custom_name": "example"},
                           {"custom_name": "example2"}]
                           },
```